manifest.yaml

```diff
@@ -1,7 +1,8 @@
 id: lnbits 
 title: "LNBits"
-version: 0.9.5.1
-release-notes: "Initial Release for embassyOS"
+version: 0.9.7
+release-notes: | 
+  * Fusion46 [Release Notes](https://github.com/lnbits/lnbits/releases/tag/0.9.6.1)
```

Comment:
Why no release notes for 0.9.7?

Reply:
very strange to me too - the release notes link in 0.9.7 links to 0.9.6.1 ... like they didn't bother making another one, just linked to the same one as the full release notes

Reply:
I could link it to https://github.com/lnbits/lnbits/releases/tag/0.9.7, but that's basically empty

Reply:
i'll put both :-)